Analysis/R/prepare_stan_input.R

```diff
@@ -213,14 +213,27 @@ prepare_stan_input <- function(
     }
   }
   
+  # ---- Da. Drop multi-year ----
+  
+  # Set admin level
+  sf_cases_resized <- sf_cases_resized %>% 
+    dplyr::mutate(admin_level = purrr::map_dbl(location_name, ~ taxdat::get_admin_level(.)))
+  
+  # Drop multi-year observations if present
+  if (drop_multiyear_adm0) {
```

Comment:
Do we need to have a call to `get_space_time_ind_speedup` here?

Reply:
That's right, good catch.